integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -157,18 +176,33 @@ impl TestReshardingParametersBuilder {
         let validators = validators.to_vec();
         let (rpcs, tmp) = tmp.split_at(num_rpcs as usize);
         let rpcs = rpcs.to_vec();
-        let (archivals, _) = tmp.split_at(num_archivals as usize);
+        let (archivals, clients_without_role) = tmp.split_at(num_archivals as usize);
         let archivals = archivals.to_vec();
 
+        if let Some(tracked_shard_schedule) = &tracked_shard_schedule {
+            assert!(clients_without_role.contains(&clients[tracked_shard_schedule.client_index]));
+            let schedule_length = tracked_shard_schedule.schedule.len();
+            assert!(schedule_length > TESTLOOP_NUM_EPOCHS_TO_WAIT as usize);
+            for i in
+                (TESTLOOP_NUM_EPOCHS_TO_WAIT - GC_NUM_EPOCHS_TO_KEEP - 1) as usize..schedule_length
+            {
+                assert_eq!(
+                    tracked_shard_schedule.schedule[i - 1],
+                    tracked_shard_schedule.schedule[i]
+                );
+            }
+        }
+
         let client_index =
             if rpcs.is_empty() { 0 } else { num_producers + num_validators } as usize;
         let client_id = clients[client_index].clone();
 
         println!("Clients setup:");
         println!("Producers: {producers:?}");
         println!("Validators: {validators:?}");
-        println!("Rpcs: {rpcs:?}, to serve requests we use client: {client_id}");
+        println!("Rpcs: {rpcs:?}");
         println!("Archivals: {archivals:?}");
+        println!("To serve requests, we use client: {client_id}");
 
```

Comment:
It might be useful to print how many extra nodes are spawned

Reply:
fixed in 66f279e879863ae6b1144cdd6ba70c595c62cf6c